ubyssey/static/src/styles/main.scss

```diff
@@ -88,6 +88,14 @@ body, html, #page-wrapper {
   -webkit-overflow-scrolling: touch;
 }
 
+span.upper-right-decorator {
+  color: $color-ub-blue-light;
+  font-size: .8em;
```

Comment:
Might look better if we make it smaller for Mobile

![image](https://user-images.githubusercontent.com/9669739/47968761-ffcdfe00-e022-11e8-89b0-a9b04ba9d524.png)

Reply:
Playing around with this, I don't see much return for effort. These new buttons will likely be removed in a month or so.